CalPatRec/src/PhiClusterFinderDiag_tool.cc

```diff
@@ -0,0 +1,121 @@
+
+///////////////////////////////////////////////////////////////////////////////
+//
+///////////////////////////////////////////////////////////////////////////////
+#include "fhiclcpp/ParameterSet.h"
+
+#include "CLHEP/Matrix/Vector.h"
```

Comment:
please, review the list of these includes.

Reply:
I've cleaned the list of includes.  [CalPatRec/src/PhiClusterFinderDiag_tool.cc](https://github.com/NamithaChitrazee/Offline/commit/6da4c6b6b23a5d9cdae1c3d40d8475aea48a731b#diff-455ed377eb30267abd879308114e4fd8d4d3456b9b0b5bee5e3110900ef18b0f)